src/style.scss

```diff
@@ -22,8 +22,8 @@
   animation-iteration-count: infinite;
   animation-name: placeHolderShimmer;
   animation-timing-function: linear;
-  background-color: var(--auro-color-brand-neutral-100);
-  background-image: linear-gradient(to right, var(--auro-color-brand-neutral-100) 5%, var(--auro-color-background-lightest) 25%, var(--auro-color-brand-neutral-100) 35%);
+  background-color: var(--auro-color-brand-gray-200);
+  background-image: linear-gradient(to right,var(--auro-color-brand-gray-200) 10%,var(--auro-color-background-lightest) 40%,var(--auro-color-brand-gray-200) 60%);
```

Comment:
@leeejune note this change as I see you referenced it in your Figma doc.

Reply:
I think I might've missed the second token "var(--auro-color-background-lightest)" would become "var(--auro-color-brand-neutral-100)".

Reply:
But not entirely sure because I don't know what it looks like in the demo!

Reply:
Demo updated

<img width="630" alt="Screen Shot 2021-11-29 at 12 30 24 PM" src="https://user-images.githubusercontent.com/181089/143938503-1f284fd9-b473-405d-9db7-fc8ad7f49caf.png">